xep-0369.xml

```diff
@@ -80,11 +85,11 @@
 
 <section1 topic='Requirements' anchor='reqs'>
   <ul>
-    <li>A user's participation in a channel persists and is not modified by the user's client going online and offline.</li>
+    <li>A user's participation in a channel may persist and is not modified by the user's client going online and offline.</li>
```

Comment:
It's not really a mode we need to introduce, it's just a matter of server policy whether or not we persist membership (or expire it after some time, or just remove it as soon as you publish an unavailable status, etc.)?